### PR TITLE
Unknown host fix

### PR DESCRIPTION
**Pull request by sanovikov71**
Change-Id: I5dbb92d5fe29651365535b49492f26d525f8ec93

### CODE REVIEW

File: app/src/main/java/org/imozerov/streetartview/network/FetchService.kt [hunk 1]
```diff
@@ -30,15 +29,20 @@ class FetchService : IntentService("FetchService") {
...
+                Log.e(TAG, "Unable to fetch data from server! ${response.code()}")
+                return
+            }
 
-        val responseJson = response.body().artworks
+            val responseJson = response.body().artworks
 
-        dataSource.insert(responseJson)
+            dataSource.insert(responseJson)
+        } catch (exception: java.net.UnknownHostException) {
```
Comment (genaforvena) [inline app/src/main/java/org/imozerov/streetartview/network/FetchService.kt]: Suggesting to handle IOException here as it the one that is thrown by Retrofit.

File: app/src/main/java/org/imozerov/streetartview/network/FetchService.kt [hunk 2]
```diff
@@ -30,15 +29,20 @@ class FetchService : IntentService("FetchService") {
...
+            Log.e(TAG, "Unknown host exception. It can be caused by turned off network connection")
```
Comment (genaforvena) [inline app/src/main/java/org/imozerov/streetartview/network/FetchService.kt]: Please log with 
`Log.w(TAG, "Unable to sync art objects with server", exception)`

File: app/src/main/java/org/imozerov/streetartview/network/FetchService.kt (hunk 1 above)
Comment (sanovikov71) [inline app/src/main/java/org/imozerov/streetartview/network/FetchService.kt]: done

File: app/src/main/java/org/imozerov/streetartview/network/FetchService.kt (hunk 2 above)
Comment (sanovikov71) [inline app/src/main/java/org/imozerov/streetartview/network/FetchService.kt]: done

**Comment by genaforvena** [comment]
:+1:

**Pull request merged**